guides/concepts/networks.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Networks
+---
+
+Stellar has two public networks: the Public Network (pubnet), which is the main decentralized
+network used throughout the world, and the Test Network (testnet), a network maintained by the
+Stellar Development Foundation that is available for developers to test their applications. For
+more in-depth information on testnet, [see our our additional documentation](test-net.md).
```

Comment:
could just make `test-net.md` a link the first time you mention it, and drop this final clause